### DISCUSSION TITLE
Clonotype analysis with already TRUST4 data

### DISCUSSION THREAD

**Discussion by pyarahmadi** [milaboratory/mixcr#1067] [Q&A]
Hi!

I have a TRUST4 barcode out data  for TCRs with these columns  "Barcode"                  "Type"                     "v_call"                  
"d_call"                   "j_call"                   "c_call"                  "cdr3"                     "cdr3_aa"                  "consensus_count"         
"sequence_id"              "cdr3_germline_similarity" "complete_vdj" .

Is there any way to find clonotypes in my dataset?

**Comment by PoslavskySV** [comment]
Hi, Do you have raw .fastq files?

**Comment by pyarahmadi** [comment]
Hi!
Thank you for your reply.
Unfortunately, no!

From: Stanislav Poslavsky ***@***.***>
Date: Thursday, April 6, 2023 at 13:49
To: milaboratory/mixcr ***@***.***>
Cc: pyarahmadi ***@***.***>, Author ***@***.***>
Subject: Re: [milaboratory/mixcr] Clonotype analysis with already TRUST4 data (Discussion #1067)

Hi, Do you have raw .fastq files?

**Comment by PoslavskySV** [comment]
Well, then you can try exporting 'complete_vdj' column into .fasta file and use MIXCR [generic preset](https://mixcr.com/mixcr/reference/overview-built-in-presets/#generic-bcr-amplicon) to analyze it. However, I would like to suggest being cautious when interpreting the results generated by TRUST: we has observed that the output from TRUST may contain numerous spurious contigs, which either has wrong sequnece or not present in the original .fastq data at all.

**Comment by pyarahmadi** [comment]
My data is like this:
<img width="390" alt="image" src="https://user-images.githubusercontent.com/130099495/230522927-1b085061-a47c-44f5-8818-224ea3a6877d.png">




Is there any way to do clonotype analysis on it?

From: Stanislav Poslavsky ***@***.***>
Date: Thursday, April 6, 2023 at 14:16
To: milaboratory/mixcr ***@***.***>
Cc: pyarahmadi ***@***.***>, Author ***@***.***>
Subject: Re: [milaboratory/mixcr] Clonotype analysis with already TRUST4 data (Discussion #1067)

Well, then you can try exporting 'complete_vdj' column into .fasta file and use MIXCR generic preset<https://mixcr.com/mixcr/reference/overview-built-in-presets/#generic-bcr-amplicon> to analyze it. However, I would like to suggest being cautious when interpreting the results generated by TRUST: we has observed that the output from TRUST may contain numerous spurious contigs, which either has wrong sequnece or not present in the original .fastq data at all.
